### ISSUE TITLE
latex ~ won't render in github

### ISSUE THREAD

**Issue by hzzyyy** [mandli/intro-numerical-methods#42]
Hi Kyle,

I find in your notebook '~' is used for empty spaces, however, it won't get rendered correctly in github (right above the second cell block):
https://github.com/mandli/intro-numerical-methods/blob/master/14_LA_iterative.ipynb

It is rendered correctly on nbviewer however:
http://nbviewer.jupyter.org/github/mandli/intro-numerical-methods/blob/master/14_LA_iterative.ipynb

The way to get around is to use \quad or \[empty space].

I also use jupyter notebook for self-study and came across your wonderful lecture notes, when you cited an issue I raised for nbviewer:
https://github.com/jupyter/nbviewer/issues/590

Cheers, 
Zhangyi

**Comment by mandli** [comment]
Yeah, I have slowly been trying to replace all the LaTeX that is not compatible.  The annoying thing is that `~` used to work too.  I am not sure what to do as I am really not thrilled with going back to replace all the `~`.  There are some other inconsistencies similar to the ones that you pointed to in the nbviewer issue as well.
